src/StandaloneModuleGraph.zig

```diff
@@ -836,7 +836,7 @@ pub const StandaloneModuleGraph = struct {
             fd,
             bun.FD.cwd(),
             bun.sliceTo(&(try std.posix.toPosixPath(temp_location)), 0),
-            bun.toFD(root_dir.fd),
+            .fromStdDir(root_dir),
```

Comment:
what if we did `.from` and made it work with std.fs.Dir, std.fs.File, std.posix.fd_t etc.